<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="infoPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createProcessButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
  background-color: #0095ff;
 border-radius: 12px;
  border-style: none;
  box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  display: inline-block;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,&quot;Liberation Sans&quot;,sans-serif;
  font-size: 20px;
  font-weight: 500;
  line-height: 1.15385;
  margin: 0;
  outline: none;
  padding: 8px .8em;
  position: relative;
  text-align: center;
  text-decoration: none;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  vertical-align: baseline;
  white-space: nowrap;
}

QPushButton:hover,
QPushButton:focus {
  background-color: #07c;
}

QPushButton:focus {
  box-shadow: 0 0 0 4px rgba(0, 149, 255, .15);
}

QPushButton:pressed {
  background-color: #0064bd;
  box-shadow: none;
}</string>
    </property>
    <property name="text">
     <string>Create Process</string>
    </property>
   </widget>
   <widget class="QComboBox" name="processCountComboBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 2px solid blue;;
  border-radius: 8px;
</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="createIdeasPushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
  background-color: #0095ff;
 border-radius: 12px;
  border-style: none;
  box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  display: inline-block;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,&quot;Liberation Sans&quot;,sans-serif;
  font-size: 20px;
  font-weight: 500;
  line-height: 1.15385;
  margin: 0;
  outline: none;
  padding: 8px .8em;
  position: relative;
  text-align: center;
  text-decoration: none;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  vertical-align: baseline;
  white-space: nowrap;
}

QPushButton:hover,
QPushButton:focus {
  background-color: #07c;
}

QPushButton:focus {
  box-shadow: 0 0 0 4px rgba(0, 149, 255, .15);
}

QPushButton:pressed {
  background-color: #0064bd;
  box-shadow: none;
}</string>
    </property>
    <property name="text">
     <string>Show Ideas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startVoatingButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
  background-color: #0095ff;
 border-radius: 12px;
  border-style: none;
  box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  display: inline-block;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,&quot;Liberation Sans&quot;,sans-serif;
  font-size: 20px;
  font-weight: 500;
  line-height: 1.15385;
  margin: 0;
  outline: none;
  padding: 8px .8em;
  position: relative;
  text-align: center;
  text-decoration: none;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  vertical-align: baseline;
  white-space: nowrap;
}

QPushButton:hover,
QPushButton:focus {
  background-color: #07c;
}

QPushButton:focus {
  box-shadow: 0 0 0 4px rgba(0, 149, 255, .15);
}

QPushButton:pressed {
  background-color: #0064bd;
  box-shadow: none;
}</string>
    </property>
    <property name="text">
     <string>Start voting</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="voatingResultPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
    <property name="text">
     <string>Virtual Ð’lackboard:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
    <property name="text">
     <string>Choose number of processes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
    <property name="text">
     <string>Results of Voting:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
